x/ibc/core/02-client/abci_test.go

```diff
@@ -58,3 +62,32 @@ func (suite *ClientTestSuite) TestBeginBlocker() {
 		prevHeight = localHostClient.GetLatestHeight().(types.Height)
 	}
 }
+
+func (suite *ClientTestSuite) TestBeginBlockerConsensusState() {
+	cs := []byte("IBC client state")
+	plan := &upgradetypes.Plan{
+		Name:                "test",
+		Height:              suite.chainA.GetContext().BlockHeight() + 1,
+		UpgradedClientState: cs,
+	}
+	store := suite.chainA.GetContext().KVStore(suite.chainA.App.GetKey(upgradetypes.StoreKey))
+	bz := suite.chainA.App.AppCodec().MustMarshalBinaryBare(plan)
+	store.Set(upgradetypes.PlanKey(), bz)
+
+	suite.T().Log("Verify that chain committed to consensus state on the last height it will commit")
```

Comment:
Is this in keeping with the rest of the IBC test structure.

Ideally we don't just have a single test that emits logs like this.